### PR TITLE
add South-Tyrolean layers

### PR DESCRIPTION
**Pull request by tyrasd**
for #1

and I've also added a parmalink component (stores map location and selected layer in url hash)

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -53,11 +68,55 @@ var attributionOsm = '<a href="https://www.openstreetmap.org/copyright">OpenStre
...
+});
+
+[
+  {id: 'DTM-2p5m_Hillshade,DTM_Hillshade_SolarTirol_3857', title: 'DTM South Tyrol'},
+  {id: 'DSM-2p5m_Hillshade,DSM_Hillshade_SolarTirol_3857', title: 'DSM South Tyrol'}
+].forEach(function(options) {
+  var layer = L.tileLayer.wms('https://geoservices.buergernetz.bz.it/geoserver/p_bz-elevation/ows', {
+    layers: options.id,
+    format: 'image/jpeg',
+    attribution: attribution
```
Comment (simon04) [inline app.js]: This attribution is wrong.

File: app.js [hunk 2]
```diff
@@ -53,11 +68,55 @@ var attributionOsm = '<a href="https://www.openstreetmap.org/copyright">OpenStre
...
     layers: options.id,
     format: 'image/jpeg',
     attribution: attribution
   });
   layers.addBaseLayer(layer, options.title);
+  allMapLayers[options.id] = layer;
 });
+
+[
+  {id: 'P_BZ_BASEMAP_TOPO', title: 'South Tyrol Base Map'},
```
Comment (simon04) [inline app.js:82]: I would have used German layer names, but I'm happy to take the English ones as well. :smile:

**Review by simon04 — CHANGES_REQUESTED** [review]
Great! Thank you for your contribution. Please fix the attribution and I'll merge.

File: app.js (hunk 1 above)
Comment (tyrasd) [reply app.js]: fixed!

File: app.js (hunk 2 above)
Comment (tyrasd) [reply app.js:82]: feel free to tweak the names. (I've chosen English since the alternative German+Italian would have been a bit long.)

**Review by simon04 — APPROVED** [review]

**Pull request merged**